geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/BinaryExportCommand.scala

```diff
@@ -0,0 +1,63 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
```

Comment:
@aheyne Ah!  Can try running
```mvn license:check``` locally?  

You might have to sort out the spacing for this file to pass.